.ci/install_cri_containerd.sh

```diff
@@ -29,9 +29,7 @@ CONTAIENRD_ARCH=$(go env GOARCH)
 cri_containerd_tarball_version=$(get_version "externals.cri-containerd.version")
 cri_containerd_repo=$(get_version "externals.cri-containerd.url")
 
-echo "Get cri_containerd version"
-cri_containerd_version_url="https://raw.githubusercontent.com/containerd/containerd/${cri_containerd_tarball_version}/vendor.conf"
-cri_containerd_version=$(curl -sL $cri_containerd_version_url | grep "github.com/containerd/cri" | awk '{print $2}')
+cri_containerd_version=$(cri_containerd_tarball_version)
```

Comment:
Should this be `cri_containerd_version="$cri_containerd_tarball_version"`
The CI run into this error:

```
03:19:00 .ci/install_cri_containerd.sh: line 32: cri_containerd_tarball_version: command not found
03:19:01 Failed at 32: cri_containerd_version=$(cri_containerd_tarball_version)
```